LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Fortran-Lang
```

Comment:
```suggestion
Copyright (c) 2020-2022 Fortran-Lang
```

Because the logo was first licensed under MIT in fortran-lang/fortran-lang.org in 2020.

Reply:
Thanks for catching this